gemrb/plugins/SDLVideo/SDL20Video.cpp

```diff
@@ -22,670 +22,668 @@
 
 #include "Interface.h"
 
-#include "GUI/Button.h"
-#include "GUI/Console.h"
-#include "GUI/GameControl.h" // for TargetMode (contextual information for touch inputs)
-
 using namespace GemRB;
 
-//touch gestures
-#define MIN_GESTURE_DELTA_PIXELS 5
-#define TOUCH_RC_NUM_TICKS 500
-
 SDL20VideoDriver::SDL20VideoDriver(void)
 {
-	assert( core->NumFingScroll > 1 && core->NumFingKboard > 1 && core->NumFingInfo > 1);
-	assert( core->NumFingScroll < 5 && core->NumFingKboard < 5 && core->NumFingInfo < 5);
-	assert( core->NumFingScroll != core->NumFingKboard );
-
 	renderer = NULL;
 	window = NULL;
-	screenTexture = NULL;
 
-	// touch input
-	ignoreNextFingerUp = 0;
-	ClearFirstTouch();
-	EndMultiGesture();
+	stencilAlphaBlender = SDL_ComposeCustomBlendMode(SDL_BLENDFACTOR_ZERO, SDL_BLENDFACTOR_ONE,
```

Comment:
Codacy found an issue: [Variable 'stencilAlphaBlender' is assigned in constructor body. Consider performing initialization in initialization list.](https://app.codacy.com/manual/gemrb/gemrb/pullRequest?prid=6310305)